app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -349,9 +349,17 @@ def rbac_tenants_list
   def rbac_tenant_delete
     assert_privileges("rbac_tenant_delete")
     tenants = []
+    projects = []
     if !params[:id] # showing a tenants list
-      ids = find_checked_items.collect { |r| from_cid(r.split("-").last) }
-      tenants = Tenant.find_all_by_id(ids)
+      ids = find_checked_items
+      recs_to_delete = Tenant.find_all_by_id(ids)
```

Comment:
yea, looks like I missed the `empty` and had `tenant` twice instead of using `tenants and projects`. think this is corrected:

```ruby
def process_tenants(recs_to_delete, tasks)
  tenants, projects = recs_to_delete.partition(&:divisible)
  process_elements(tenants, Tenant, task, "Tenant", "name", :display_desc => false, :display_model => "Tenant") unless tenants.empty?
  process_elements(projects, Tenant, task, "Project", "name", :display_desc => false, :display_model => "Project) unless projects.empty?
end
```